packages/web3/src/transaction/utils.test.ts

```diff
@@ -17,115 +17,295 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
 import { getSigner, getSigners } from '@alephium/web3-test'
-import { groupIndexOfTransaction } from './utils'
+import { groupIndexOfTransaction, waitForTxConfirmation } from './utils'
 import { ONE_ALPH, TOTAL_NUMBER_OF_GROUPS } from '../constants'
 import { bs58, hexToBinUnsafe } from '../utils'
 import { unsignedTxCodec } from '../codec'
 import { groupOfAddress } from '../address'
 import { blake2b } from 'blakejs'
 import { web3 } from '../'
 import { PrivateKeyWallet } from '@alephium/web3-wallet'
+import { node } from '../api'
 
 describe('transaction utils', () => {
   beforeAll(() => {
     web3.setCurrentNodeProvider('http://127.0.0.1:22973', undefined, fetch)
   })
 
-  it('should calculate the group of transfer transaction', async () => {
```

Comment:
Why do we remove some of the existing tests?

Reply:
when I tried testing the file all the test failed. came to the realization that I needed to connect to the Alephium devnet and that has been a strucgle

Reply:
I will love your assistance on how to successfully connect to the devnet

Reply:
You can run the devnet by running `docker-compose up -d` under the `docker` directory. After that the test case should be able to connect to devnet.